packages/daemon/src/mail.js

```diff
@@ -75,6 +75,13 @@ export const makeMailboxMaker = ({
       return controller.terminator.terminate();
     };
 
+    const list = () => harden(petStore.list());
+
+    const listSpecial = () => harden(Object.keys(specialNames).sort());
+
+    const listAll = () =>
+      harden([...Object.keys(specialNames).sort(), ...petStore.list()]);
```

Comment:
Ah, they do work. Let’s just make `list()` have the behavior of `listAll()` and not have `listSpecial()`.  Sound good?